webviz_subsurface/_private_plugins/tornado_plot.py

```diff
@@ -394,44 +407,44 @@ def tornado_plot(
     df = sort_by_max(df)
     # Return tornado data as Plotly figure
 
-    return {
-        "data": [
-            dict(
-                type="bar",
-                y=df["sensname"],
-                x=df["low"],
-                name="low",
-                customdata=df["low_reals"],
-                hovertext=[
-                    f"Case: {label}<br>True Value: {val:.2f}<br>Realizations:"
-                    f"{min(reals) if reals else None}-{max(reals) if reals else None}"
-                    for label, val, reals in zip(
-                        df["low_label"], df["true_low"], df["low_reals"]
-                    )
-                ],
-                hoverinfo="x+text",
-                orientation="h",
-                marker=dict(color="rgb(235, 0, 54)"),
-            ),
-            dict(
-                type="bar",
-                y=df["sensname"],
-                x=df["high"],
-                name="high",
-                customdata=df["high_reals"],
-                hovertext=[
-                    f"Case: {label}<br>True Value: {val:.2f}<br>Realizations:"
-                    f"{min(reals) if reals else None}-{max(reals) if reals else None}"
-                    for label, val, reals in zip(
-                        df["high_label"], df["true_high"], df["high_reals"]
-                    )
-                ],
-                hoverinfo="x+text",
-                orientation="h",
-                marker=dict(color="rgb(36, 55, 70)"),
-            ),
-        ],
-        "layout": {
+    plot_data = [
+        dict(
+            type="bar",
+            y=df["sensname"],
+            x=df["low"],
+            name="low",
+            customdata=df["low_reals"],
+            hovertext=[
+                f"Case: {label}<br>True Value: {val:.2f}<br>Realizations:"
+                f"{min(reals) if reals else None}-{max(reals) if reals else None}"
+                for label, val, reals in zip(
+                    df["low_label"], df["true_low"], df["low_reals"]
+                )
+            ],
+            hoverinfo="x+text",
+            orientation="h",
+        ),
+        dict(
+            type="bar",
+            y=df["sensname"],
+            x=df["high"],
+            name="high",
+            customdata=df["high_reals"],
+            hovertext=[
+                f"Case: {label}<br>True Value: {val:.2f}<br>Realizations:"
+                f"{min(reals) if reals else None}-{max(reals) if reals else None}"
+                for label, val, reals in zip(
+                    df["high_label"], df["true_high"], df["high_reals"]
+                )
+            ],
+            hoverinfo="x+text",
+            orientation="h",
+        ),
+    ]
+    layout = {}
+    layout.update(plotly_theme["layout"])
```

Comment:
...perhaps enables us to in the different containers/plugins to simplify these two lines into:
```
layout = self.theme.plotly_theme["layout"]
```
Not tested though.